README.md

```diff
@@ -183,6 +182,14 @@ For example:
 set -g @thumbs-command 'echo -n {} | pbcopy'
 ```
 
+OSC52:
+
+Depends on tmux git (at least fc9cdc)  and on `set -s set-clipboard on|external` in tmux config.
```

Comment:
For old versions of tmux, can we suggest a command like?:

```
set -g @thumbs-command 'tmux set-buffer {} | echo -e "\033]52;c;$(base64 <<< {})\a"'
```

Reply:
Can we point `fc9cdc` to the original commit, please?

Reply:
> For old versions of tmux, can we suggest a command like?:
> 
> ```
> set -g @thumbs-command 'tmux set-buffer {} | echo -e "\033]52;c;$(base64 <<< {})\a"'
> ```
won't work :( when doing `tmux run blah` `blah` does not run in terminal see: https://github.com/tmux/tmux/issues/2363#issuecomment-685432373

> Can we point fc9cdc to the original commit, please?
>

I have no idea how to do that but let me figure it out

Reply:
> won't work :( when doing tmux run blah blah does not run in terminal see: tmux/tmux#2363 (comment)
Ouch true :sweat_smile: 

> I have no idea how to do that but let me figure it out

I can add it later